src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/TextBlockTests/TextBlockTests.cs

```diff
@@ -457,7 +457,7 @@ public void When_TextTrimming_Is_Set_Then_Ellipsis_Is_Used()
 
 		[Test]
 		[AutoRetry]
-		[ActivePlatforms(Platform.Browser | Platform.Android)] // Test timed-out on iOS
```

Comment:
This one is funny.

Android = 1, Browser = 2 (0b0001 | 0b0010 = 0b0011 = 3)

The value 3 isn't a valid Platform at all! So, I think this test was *never* run!